src/reconcile/rec_visibility.c

```diff
@@ -430,6 +431,7 @@ __timestamp_no_ts_fix(WT_SESSION_IMPL *session, WT_TIME_WINDOW *select_tw)
 
```

Comment:
I think we can assert if we have prepared stop, stop_prepare_ts must be larger than or equal to start ts.

Reply:
sorry which line did you refer to here?

Reply:
before line 425 I think.